app/views/layouts/map.html.erb

```diff
@@ -46,12 +46,17 @@
 
     <% unless @user %>
       <div class="welcome">
-        <h2><a><span class="icon close"></span></a><%= t 'layouts.intro_header' %></h2>
+        <h2><%= t 'layouts.intro_header' %></h2>
+        <div class="close-wrap"><span class="icon close"></span></div>
```

Comment:
Good question - I added `close-wrap` to give the close button a larger touch target.
We use `class="icon close"` a lot of places to draw those close icons.. I didn't want to change all of them, just the ones on the Welcome and the Banner divs.

If we just absolutely position the close icon on top of the banner without the wrapping div, we get this:
<img width="516" alt="screenshot 2016-06-21 14 39 35" src="https://cloud.githubusercontent.com/assets/38784/16241979/513218f4-37be-11e6-8046-32c9c60641a9.png">

The problem with that is: if the user misses clicking on that little 'X', their click will go to the banner div and it will send them to the other website.


With the `close-wrap`, the user has a much larger target to click, and won't accidentally click the banner.:

<img width="459" alt="screenshot 2016-06-21 14 37 58" src="https://cloud.githubusercontent.com/assets/38784/16241987/56d1b210-37be-11e6-9071-40f4724442d3.png">